students/GuyCChan/session09/test_objOriMailRoom.py

```diff
@@ -0,0 +1,226 @@
+from objOriMailRoom import Donor, DonorCollection, make_report, letters_to_donors, test_data
+import pytest
+
+# Tests for Donor class.
+def test_init():
+    db = donors_db()
+    assert len(donors_db) == 0
+
+def test_Donor_in_db():
+    donor = donor("Mike May")
+    assert donor.name == "Mike May"
+    assert donor.donation is None
+
+def test_add_donation():
+    add_donation = self.donations("100.00")
```

Comment:
Donor.donations is a method? that takes a string? probably not what you want. and there is not self defined here...

this should probably look something like:

```
def test_add_donation():
    donor = Donor("Karl Marx")
    donor.add_donation(100)
    assert donor.last_donation == 100
```